HttpFoundationFactoryInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Symfony\Bridge\PsrHttpMessage;
+
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Creates Symfony Request and Response from PSR-7 ones.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+interface HttpFoundationFactoryInterface
```

Comment:
@stof I agree but it was to be consistent with the other side, and the other side needs an interface.